### DISCUSSION TITLE
【灌水】那个绿毛大姐头

### DISCUSSION THREAD

**Discussion by JustOldJohn** [share121/inter-knot#1561] [其他]
![IMG_20240812_004901](https://github.com/user-attachments/assets/e11e3d41-e004-43d2-ab17-042daeef89fa)

下午打工下班的时候路过光映广场那边，看到有几个穿着很前卫的人从超市走出来，里面有个浅绿短发有点高的女生超帅的！看起来像是大姐头，我想她要是笑起来会很可爱吧，最后他们要走的时候（竟然是个车队）我还是忍不住远远的拍了一张（封面）。希望以后还能见到她～

**Comment by gaojiaobei** [comment]
虾头

**Comment by kaijinti** [comment]
她好像是卡吕冬之子的老大来着？你也许可以去问问狡兔屋的比利怎么才能加入她们，只要你给够妮可丁尼就行
